2023-08-08 12:42:16,067 INFO    MainThread:8263 [wandb_setup.py:_flush():76] Current SDK version is 0.15.4
2023-08-08 12:42:16,067 INFO    MainThread:8263 [wandb_setup.py:_flush():76] Configure stats pid to 8263
2023-08-08 12:42:16,067 INFO    MainThread:8263 [wandb_setup.py:_flush():76] Loading settings from /home/bugsie/.config/wandb/settings
2023-08-08 12:42:16,067 INFO    MainThread:8263 [wandb_setup.py:_flush():76] Loading settings from /home/bugsie/PycharmProjects/FiberPhotometryAnalyzer/wandb/settings
2023-08-08 12:42:16,067 INFO    MainThread:8263 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2023-08-08 12:42:16,067 INFO    MainThread:8263 [wandb_setup.py:_flush():76] Applying setup settings: {'_disable_service': False}
2023-08-08 12:42:16,067 INFO    MainThread:8263 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program_relpath': 'train_behavior.py', 'program': '/home/bugsie/PycharmProjects/FiberPhotometryAnalyzer/train_behavior.py'}
2023-08-08 12:42:16,067 INFO    MainThread:8263 [wandb_init.py:_log_setup():507] Logging user logs to /home/bugsie/PycharmProjects/FiberPhotometryAnalyzer/wandb/run-20230808_124216-9u610nni/logs/debug.log
2023-08-08 12:42:16,067 INFO    MainThread:8263 [wandb_init.py:_log_setup():508] Logging internal logs to /home/bugsie/PycharmProjects/FiberPhotometryAnalyzer/wandb/run-20230808_124216-9u610nni/logs/debug-internal.log
2023-08-08 12:42:16,067 INFO    MainThread:8263 [wandb_init.py:init():547] calling init triggers
2023-08-08 12:42:16,067 INFO    MainThread:8263 [wandb_init.py:init():554] wandb.init called with sweep_config: {}
config: {}
2023-08-08 12:42:16,067 INFO    MainThread:8263 [wandb_init.py:init():591] wandb.init() called when a run is still active
2023-08-08 12:42:18,341 INFO    MainThread:8263 [wandb_run.py:_config_callback():1283] config_cb None None {'input_shape': [1, 1000, 8], 'embedding_shape': [1, 32], 'learning_rate': 1e-06, 'num_layers': 3, 'num_heads': 8}
2023-08-08 12:42:25,897 WARNING MsgRouterThr:8263 [router.py:message_loop():77] message_loop has been closed
